docs/openrpc.json

```diff
@@ -0,0 +1,917 @@
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{64}$"
+          }
+        },
+        {
+          "name": "actions",
+          "summary": "Set of deposits and withdrawals to be performed to/from the contract",
+          "required": true,
+          "schema": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/NanoContractAction"
+            }
+          }
+        },
+        {
+          "name": "args",
+          "summary": "Arguments of method being called",
+          "required": true,
+          "schema": {
+            "type": "array"
+          }
+        },
+        {
+          "name": "push_tx",
+          "description": "If true, wallet application signs transaction and pushs it to network. If false, wallet application returns signed transaction to dApp (ready to be pushed).",
+          "required": true,
+          "schema": {
+            "type": "boolean"
+          }
+        }
+      ],
+      "result": {
+        "name": "SendNanoContractTxResponse",
+        "schema": {
+          "type": "object",
+          "oneOf": [
+            {
+              "description": "If push_tx parameter was false, wallet application returns transaction encoded in hexadecimal, ready to be pushed by any one."
+              "$ref": "#/components/schemas/TransactionHex"
+            },
+            {
+              "description": "If push_tx parameter was true, wallet application returns the transaction already registered in the blockchain."
+              "$ref": "#/components/schemas/NanoContract"
+            }
+          ]
+        }
+      },
+      "errors": [
+        {
+          "$ref": "#/components/errors/NCBadRequest"
+        },
+        {
+          "$ref": "#/components/errors/PromptRejectedError"
+        },
+        {
+          "$ref": "#/components/errors/SendNanoContractTxFailure"
+        }
+      ]
+    },
+    {
+      "name": "htr_createToken",
+      "summary": "Requests user to create token",
+      "description": "Provides user the deailts for a token creation. dApp provides wallet application with data that comprises a token creation. Wallet application assembles and signs transaction, and then pushes it to network or returns it to dApp.",
+      "paramStructure": "by-name",
+      "params": [
+        {
+          "name": "name",
+          "required": true,
+          "schema": {
+            "description": "UTF-8 characters.",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 30
+          }
+        },
+        {
+          "name": "symbol",
+          "required": true,
+          "schema": {
+            "description": "UTF-8 characters.",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 5
+          }
+        },
+        {
+          "name": "amount",
+          "required": true,
+          "schema": {
+            "type": "number"
+          }
```

Comment:
OutputValue schema?

Reply:
Suggestion: keep as is:
Reason:
- In fact, these are values ​​that will always be the same, but they are different abstractions.
- The outputvalue refers specifically to the value of an output.
- Here, it refers to the number of tokens to be created.
- For example, this follows the same parameter description pattern that we already use in the headless "create token" API: https://docs.hathor.network/references/headless-wallet/http-api/create-token

Reply:
They are not different abstractions, this amount is the amount of the output of the newly created token, so it's the same meaning.

Reply:
The thing is, the schema you pointed out is for a parameter, not a property of the result. 
It's the parameter that defines how many tokens will be created, not the amount of tokens that were created. At this point, there is no output yet. There is only the data for the amount of tokens the DApp wants the wallet to create.

Reply:
OK: change made:
- Replaced number by reference to `OutputValueType` schema.